### DISCUSSION TITLE
Signups false wont work

### DISCUSSION THREAD

**Discussion by Nollknolle** [dani-garcia/vaultwarden#2709] [Q&A]
Hello,
i added the following env variable to my container but now i can create accounts too.
do i missed something?

SIGNUPS_ALLOWED=false

**Comment by BlackDex** [comment]
See: https://github.com/dani-garcia/vaultwarden/wiki/Disable-registration-of-new-users

**Reply by Nollknolle** [reply]
I did it exactly as described there but it does not work.

**Reply by BlackDex** [reply]
What doesn't work exactly?
Did you restart/recreated the vaultwarden container after changing the value?
What does the /admin interface in the diagnostics panel tell you.
Is there a config.json which overrides the settings (Which should be shown during the startup or in the logs of the container).

**Reply by Nollknolle** [reply]
We recreated it, yes.
We have not another config
where can I find the diagnostic?

**Reply by BlackDex** [reply]
Diagnostics can be found in the `/admin` interface.
